web/client/reducers/catalog.js

```diff
@@ -129,6 +130,8 @@ function catalog(state = {
         return set("newService.url", action.url, state);
     case CHANGE_AUTOLOAD:
         return set("newService.autoload", action.autoload, state);
+    case CHANGE_SERVICE_FORMAT:
```

Comment:
other parts has test, this should be have it too.

I know testing reducers by themself, when this are so simple like this, is a little boring and seems useless. But we need them to ensure to keep the same interface. 

editor's note: In other places, to make these tests more useful I used to apply to use actionCreators and selector, so I test the integration between them and so all the state cicle (event creation --> state mutation --> state slice getter). If you prefer, you can use this approch